installers/icons/README.md

```diff
@@ -12,9 +12,15 @@ To generate ico files:
 
     icotool -c -o 256x256.ico 256x256.png
 
-Currently, the Windows installer uses only the 64x64 icon.
+To Merge all icons into a single bundle:
+
+    nix run nixpkgs.imagemagick
```

Comment:
This doc now uses two different commands for starting a nix shell. Old command (`nix-shell`) and new command (`nix run`). Best to use one or the other or it will cause confusion.

Reply:
@rvl may I ask you to update this? @cleverca22 will not come online soon and I would like to merge this PR as it is our last one for the 0.11.1 release.

Reply:
@rvl I agree that other ICO files are not used but I would like to keep them in the repo as they might be useful in the future.